client/landing/stepper/declarative-flow/internals/steps-repository/sensei-setup/style.scss

```diff
@@ -0,0 +1,139 @@
+@import "@wordpress/base-styles/breakpoints";
+
+$sensei-color: #43af99;
+
+.sensei-setup {
+	.progress-bar__progress {
+		background-color: $sensei-color;
+	}
+
+	.css-1l7urtg-Container {
+		height: auto;
+	}
```

Comment:
Just as I mentioned in another PR. You cannot rely on this class. It's auto generated and will not be the same in production builds.